inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -35,8 +35,8 @@ def application_add(args: argparse.Namespace) -> str:
 
     manifest += '<repo><repos>'
 
-    for source in args.sources:
-        manifest += '<source_pkg>' + source + '</source_pkg>'
+    source_tags = [f'<source_pkg>{source}</source_pkg>' for source in args.sources]
```

Comment:
Ensure that the refactored list comprehension and string join operations do not alter the expected XML structure, particularly with respect to whitespace and element ordering. [important]